lib/semantics/resolve-names.cc

```diff
@@ -5044,6 +5070,9 @@ bool ResolveNamesVisitor::Pre(const parser::ImportStmt &x) {
       return false;
     }
     break;
```

Comment:
It would be nice if you add which part of the standard are you implementing!
It is not clause C1614.

Reply:
There is no constraint C1614.  The exclusion of `IMPORT` from `BLOCK DATA` is implied by C1415.  I added a comment.

Reply:
Ops! Sorry. I mean C1416.
A type declaration statement in  ablock-data specification-part shall not contain ALLOCATABLE, EXTERNAL,or BIND attribute specifiers.
But, yes I remember that IMPORT cannot be inside BLOCK DATA. If I am correct it is in another part of the standard